src/pages/friend.js

```diff
@@ -1,5 +1,160 @@
+  "Doodle Race",
+  "Cards Against Humanity",
+  "Trivia Games",
+  "Chess",
+  "Code Names",
+  "Hangman",
+  "Checkers",
+];
+
+const current_user_id = 2; //localStorage.getItem("current");
+
+const getFriend = async () => {
+  var response = await GET("/friends/" + current_user_id);
+  return response;
+};
+
 function Friend() {
-  return <h1>Friend</h1>;
+  const [friendData, setFriendData] = useState({ email: "", id: "" });
+  const [notPaired, setNotPaired] = useState(false);
+
+  useEffect(() => {
+    getFriend().then((response) => setFriendData(response));
+  }, [notPaired]);
+
+  useEffect(() => {
+    if (friendData.id === current_user_id) {
+      setNotPaired(true);
+    }
+  }, [notPaired, friendData]);
+
+  return (
+    <div>
+      {notPaired ? (
+        <div>
+          <div className="absolute inset-0 z-0">
+            <img
+              className="relative z-10 h-1/1 w-full object-cover"
+              src={Background}
+              alt="Universe Background"
+            />
+          </div>
+
+          <div className="relative z-10">
+            <NavBar />
+            <div className="flex">
+              <div className="w-2/3 h-full">
+                <div className="text-white ml-72 mt-20 text-5xl font-bold">
+                  Vacation week!
+                </div>
+
+                <div className="w-[50%] ml-72 mt-16 text-base text-white">
+                  Yey, look at you! So lucky, this week you’ve got a little
+                  vacation from this task. Now, when you have more time, make
+                  sure you answer the question of this week and relax. See you
+                  next week, we’ve got someone very cool in store for you.{" "}
+                </div>
+                
+                <img
+                  className="ml-96 mt-12 h-3/5 relative z-10"
+                  src={AstronautChilling}
+                  alt="AstronautChilling"
+                />
+              </div>
+
+              <div className="w-1/3 h-[1080px] bg-bkg-purple flex-shrink-0">
+                <div className="text-white mt-[30px] text-center text-[32px] font-bold">
+                  Suggested Activities
+                </div>
+
+                <div className="grid gap-8 justify-center mt-12">
+                  {activity.map((activity) => (
+                    <button class="w-72 h-16 bg-button-gray text-white font-bold rounded-[27px] shadow-md">
+                      {activity}
+                    </button>
+                  ))}
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      ) : (
+        <div>
+          <div className="absolute inset-0 z-0">
+            <img
+              className="relative z-10 h-1/1 w-full object-cover"
+              src={Background}
+              alt="Universe Background"
+            />
+          </div>
+
+          <div className="relative z-10">
+            <NavBar />
+            <div className="flex">
```

Comment:
Unless I'm missing something, the part of the UI that changes depending on being paired is inside this `div`. So it would be better if you had the ternary `{ notPaired ? ... : ... }` here, instead of wrapping everything. That way you don't repeat as much code.